app/screens/home/tab_bar/account.tsx

```diff
@@ -19,22 +20,32 @@ type Props = {
 }
 
 const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
-    selected: {
+    container: {
         borderWidth: 2,
-        borderColor: theme.buttonBg,
         borderRadius: 20,
     },
+    focused: {
+        borderColor: theme.buttonBg,
+    },
+    unfocused: {
+        borderColor: changeOpacity(theme.centerChannelColor, 0.48),
```

Comment:
Sanity check: Does this border color work with all our default themes?

Reply:
It actually is the same for a few other tab_bar icons, and others.  Perhaps I should create another PR to clean up with a helper function instead.

```
changeOpacity(theme.centerChannelColor, 0.48)
```

![changeOpacity-usage](https://github.com/user-attachments/assets/e215f65a-4be5-4e04-8502-b7f226a8672e)